popup.html

```diff
@@ -1,32 +1,32 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
-    <link rel="icon" href="assets/icon.png" />
-    <link rel="stylesheet" href="style.css" />
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
-    <title>No More Domains</title>
-    <style>
-      </style>
-  </head>
+  <link rel="icon" href="assets/icon.png" />
+  <link rel="stylesheet" href="style.css" />
 
-  <body>
-    <div class="title-block">
-      <h1 style="margin-bottom: 0.2em">No More Domains</h1>
-      <p style="margin-bottom: 1.75em; margin-top: 0.2em">
-        Stop registering new domains.
-      </p>
-    </div>
-    <div class="power-switch" style="margin: 3px;">
-      <label class="switch">
-        <input type="checkbox"  id="toggleButton" checked>
-        <span class="slider round"></span>
-      </label>
-    </div>
-    <script src="blockpage.js" type="text/javascript"></script>
-    <script src="popup.js" type="text/javascript"></script>
-  </body>
-</html>
+  <title>No More Domains</title>
+</head>
+
+<body>
+  <div class="title-block">
+    <h1 style="margin-bottom: 0.2em">No More Domains</h1>
```

Comment:
why remove some inline css but not all?

Reply:
<div class="power-switch" style="margin: 3px;">